events/freshers-selection.md

```diff
@@ -0,0 +1,118 @@
+* What are you learning these days?
+* What other societies/clubs/fests are you a part of or you want to be? How long have you been there? What are your opinions about them?
+* Write your skills (Languages, frameworks, design skills, etc.)
+* Why do you want to join KOSS? What are your expectations?
+* Enlist all of your interactions with the KOSS community over the year
+* Any comments?
+
+### Interview Round
+
+We form a few panels consisting of 3-5 people. Each panel should have at least one advisor. The interviewers evaluate them during the conversation. If a candidate does not have any projects done so far, a task must be given to check code quality.
+
+## Panel Discussion and the release of results
+
+We all meet a few days after the interview round and discuss each potential candidate. A Physical meeting is required before finalizing the list of new members.
+
+Also see [Onboarding/Offboarding](/docs/community/onboarding-offboarding)
+
+## Guidelines
+
+* Have a physical meeting to discuss the selection of freshers every year.
+* Make sure the form reaches all the freshers. Good marketing is recommended.
+* Make the students comfortable during the interview. Freshers can get nervous and you may not get to know them completely.
+* For on-spot tasks, judge by their effort.
+* Give the interviewees an experience to remember. Talk politely and be curious in them.
+* It is important that the panelists write detailed comments for every candidate after their interview. This helps in judging them later.
+* Do not share the Sheets (responses, interview comments, etc.) with KOSS internet accounts. Maintain them on personal accounts. There is no need to archive them.
+* Do not share Sheets with the link (use email addresses to add collaborators). Limit possibility of the new members viewing the sheets in the future.
+* Do not discuss selections on public channels on Slack or Google Group.
+
+### Red Flags
+
+**What do we mean by Red Flags?**
+
+> If you spot a red flag, you say No at the very moment. No second thoughts or arguments.
+> The Red flags listed below do not mean they are bad qualities as a programmer or as a human being. They only mean that the person is not a cultural fit for KOSS.
+
+* Mentions disinterest in Teaching.
+* Mentions disinterest in Learning.
+* No idea what Open Source is (little knowledge is not a red flag)
+* Highly involved in Competitive programming (as a sport)
+* “Let us join just another society”-kind of attitude
+* "I want to learn coding/data". Thinks KOSS is a society which teaches members to code.
+* Mentions “Google Summer of Code” as the primary reason to join KOSS
+* Member of a society/club/fest with high involvement (ACell/KTJ/SF/E-Cell)
+
+### Positive points
+
+* Participated in KWoC or Summit
+* Follows KOSS, already in touch with any of its members
+* Tried tools/frameworks with a mindset to explore
+* Passion
+* Self-confidence
+
+### Negative points
```

Comment:
Maybe we can make the difference more clear between `Negative points` and `Red Flags`.